misaligned.spec.js

```diff
@@ -0,0 +1,45 @@
+const {expect} = require('chai');
+const  print_color_map = require('./misaligned.js');
+// Import the print_color_map function
+// Redirect console.log to a custom function
+const originalLog = console.log;
+const capturedLogs = [];
+
+console.log = function (...args) {
```

Comment:
Bold method to test via the console :+1: 
Good. It also outputs it for manual visual inspection.

Reply:
Thank you :)